credits.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+	<title>
+		Left Shark | I do what I want
+	</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link href='https://fonts.googleapis.com/css?family=Delius+Unicase' rel='stylesheet' type='text/css'>
+	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
+</head>
+<body>
+
+
+	<img src="images/profile.svg">
+
+	<h1>Left Shark</h1>
+
+	<h2>Dance Like Nobody's Watching</h2>
+
+	<nav>
+		<a href="index.html">go left</a>
+		<a href="resume.html">other left</a>
+		<!--  <a href="credits.html">credits</a> -->
+	</nav>
+
+		<section>
+
+			<img src="images/meme.png">
+
+			<h3>CREDITS</h3>
+		
+			<p>ClassyCardsCreative on Etsy for illustration of left shark</p>
+			<p>UrbanDictionary for definitions on homepage</p>
+			<p>SBNation for movie posters</p>
+
+		</section>
+
```

Comment:
:hammer: While this won't affect the page layout, it would be best practice to remove the unnecessary empty lines. Use a single empty line to separate sections of your code for visual purposes.